website/pages/en/index.js

```diff
@@ -139,6 +139,7 @@ const Templates = props => (
       {
         content: "<div class='typeset'>Cicero templates are composed of three elements: the Template Grammar (the natural language text for the template), the Template Model (the data model that backs the template), the Logic (the executable business logic for the template). When combined these three elements allow templates to be edited, analyzed, queried and executed.</div>",
         image: imgUrl('../docs/assets/template.png'),
+        imageAlt: "Cicero templates composition using three elements: Model ,Grammer ,Logic",
```

Comment:
Please fix the English. Also I think a more concise alt text would be great, @Michael-Grover ?